app/assets/javascripts/pageflow/editor/views/model_thumbnail_view.js

```diff
@@ -17,24 +17,37 @@ pageflow.ModelThumbnailView = Backbone.Marionette.View.extend({
   },
 
   update: function() {
-    var file = this.model && this.model.thumbnailFile();
-
-    if (this.fileThumbnailView && this.currentFileThumbnail == file) {
-      return;
+    if (this.model) {
+      if (_.isFunction(this.model.thumbnailFile)) {
+        var file = this.model && this.model.thumbnailFile();
+
+        if (this.thumbnailView && this.currentFileThumbnail == file) {
+          return;
+        }
+
+        this.currentFileThumbnail = file;
+
+        this.newThumbnailView = new pageflow.FileThumbnailView({
+          model: file,
+          className: 'thumbnail file_thumbnail',
+          imageUrlPropertyName: this.options.imageUrlPropertyName
+        });
+      }
+      else {
+        this.newThumbnailView = new pageflow.StaticThumbnailView({
+          model: this.model
+        });
```

Comment:
The `StaticThumbnailView` is recreated each time update is called, which happens whenever the model configuration changes. If we want to ensure that that the current `model.thumbnailUrl()` is used after a configuration change, `StaticThumbnailView` could listen to that event itself and call its own update method.